llvm/test/CodeGen/NVPTX/inline-asm-b128-test1.ll

```diff
@@ -35,11 +35,10 @@ define void @test_b128_input_from_load(ptr nocapture readonly %data) {
 ; CHECK-NEXT:  // %bb.0:
 ; CHECK-NEXT:    ld.param.u64 %rd2, [test_b128_input_from_load_param_0];
 ; CHECK-NEXT:    cvta.to.global.u64 %rd3, %rd2;
-; CHECK-NEXT:    ld.global.u64 %rd4, [%rd3+8];
-; CHECK-NEXT:    ld.global.u64 %rd5, [%rd3];
-; CHECK-NEXT:    mov.b128 %rq1, {%rd5, %rd4};
+; CHECK-NEXT:    ld.global.v2.u64 {%rd4, %rd5}, [%rd3];
```

Comment:
^^^ This is where we could've loaded .b128 directly. Probably makes no difference on the SASS, though, just nicer looking PTX.